src/app.js

```diff
@@ -143,7 +143,11 @@ router.use(wizard(steps, fields, wizardOptions));
 
 router.use((err, req, res, next) => {
   logger.get().error("Error caught by Express handler - redirecting to Callback with server_error", {err});
-  next(err);
+	const REDIRECT_URI = req.session?.authParams?.redirect_uri;
+	if (REDIRECT_URI) {
+		next(err);
+		router.use(commonExpress.lib.errorHandling.redirectAsErrorToCallback);
```

Comment:
What does this do?

Reply:
Takes the user back to IPV core by calling the callback() when the express app encounters any error